conjur/login/login_logic.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginLogic module
+
+This module is the business logic for writing user-specific configuration
+information to the user's machine
+"""
+
+# Builtins
+import logging
+
+# Internals
+import conjur.constants
+from conjur.endpoints import ConjurEndpoint
+from conjur.http import invoke_endpoint, HttpVerb
+
+class LoginLogic:
+    """
+    LoginLogic
+
+    This class holds the business logic for populating the
+    netrc configuration details needed to login to Conjur
+    """
+    credentials = None
+
+    def __init__(self, credentials):
```

Comment:
it seems that the credentials if the `CredentialsFromFile` which can be replaced with similar objects that implement the `save`/`load` interface.
The internal name is, though, misleading.
Call it `credentials_storage` to better explain its purpose.